### ISSUE TITLE
false positive on exploit test

### ISSUE THREAD

**Issue by mikevanwinkle** [pantheon-systems/wp_launch_check#3]
```
--------------------------------------------------------------------------------
PROBABLE EXPLOITS: (Looking for exploited files.)
--------------------------------------------------------------------------------
Result: Found 1 files that contain likely exploits
	-> plugins/gravityforms/common.php [ Line 1, Match: '<?php
if(!class_exists('GFForms')){
    die(
```

**Comment by davidciplex** [comment]
Any chance this will be resolved soon?

**Comment by nstielau** [comment]
We’re looking at how to address this. Pull Requests welcome!

On Thu, Apr 9, 2015 at 11:56 AM, davidciplex <notifications@github.com>
wrote:

> Any chance this will be resolved soon?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pantheon-systems/wp_launch_check/issues/3#issuecomment-91327381>
> .
>



-- 
Nick Stielau
Director of Engineering
Pantheon Systems

**Comment by kibra** [comment]
Fixed by #14
